math/src/elliptic_curve/short_weierstrass/curves/bls12_381/curve.rs

```diff
@@ -1,13 +1,25 @@
 use super::field_extension::{BLS12381PrimeField, Degree2ExtensionField};
+use crate::cyclic_group::IsGroup;
 use crate::elliptic_curve::short_weierstrass::point::ShortWeierstrassProjectivePoint;
 use crate::elliptic_curve::traits::IsEllipticCurve;
+use crate::unsigned_integer::element::*;
 use crate::{
     elliptic_curve::short_weierstrass::traits::IsShortWeierstrass, field::element::FieldElement,
 };
 
 pub type BLS12381FieldElement = FieldElement<BLS12381PrimeField>;
 pub type BLS12381TwistCurveFieldElement = FieldElement<Degree2ExtensionField>;
 
+/// Cube Root of Unity in G1.
+pub const CUBE_ROOT_OF_UNITY_G1: U384 = U384::from_hex_unchecked(
+    "5f19672fdf76ce51ba69c6076a0f77eaddb3a93be6f89688de17d813620a00022e01fffffffefffe",
+);
+/// Cube Root of Unity in G2.
+pub const CUBE_ROOT_OF_UNITY_G2: U384 = U384::from_hex_unchecked("1a0111ea397fe699ec02408663d4de85aa0d857d89759ad4897d29650fb85f9b409427eb4f49fffd8bfd00000000aaac");
```

Comment:
Same comments as above (comment is unclear and make it `FieldElement<Degree2ExtensionField>`).

This is not used in your code. Its used though in `pairing.rs` ([here](https://github.com/lambdaclass/lambdaworks/blob/9825074e9269326bea93b788540b3bac7cde4056/math/src/elliptic_curve/short_weierstrass/curves/bls12_381/pairing.rs#L183)). So, I would leave it here and import it in `pairing.rs`